### PR TITLE
hip stack: add 5.6.0

### PR DESCRIPTION
**Pull request by dguibert**

### CODE REVIEW

File: var/spack/repos/builtin/packages/hsa-rocr-dev/package.py [hunk 1]
```diff
@@ -145,6 +147,7 @@ class HsaRocrDev(CMakePackage):
         "5.4.3",
         "5.5.0",
         "5.5.1",
+        "5.6.0",
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/hsa-rocr-dev/package.py:151]: Should `5.4.4` also be added above line 148?

File: var/spack/repos/builtin/packages/hsa-rocr-dev/package.py [hunk 2]
```diff
@@ -24,8 +24,10 @@ class HsaRocrDev(CMakePackage):
     libraries = ["libhsa-runtime64"]
 
     version("master", branch="master")
+    version("5.6.0", sha256="30875d440df9d8481ffb24d87755eae20a0efc1114849a72619ea954f1e9206c")
     version("5.5.1", sha256="53d84ad5ba5086ed4ad67ad892c52c0e4eba8ddfa85c2dd341bf825f4d5fe4ee")
     version("5.5.0", sha256="8dbc776b56f93ddaa2ca38bf3b88299b8091de7c1b3f2e481064896cf6808e6c")
+    version("5.4.4", sha256="6a0c6300ccf812b6341097e999394389a6343d8fcc92740a8636f61631e2d276")
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/hsa-rocr-dev/package.py:30]: Is it safe to assume that this version does not also need to be added to the `hip` and other packages here that don't have it?

File: var/spack/repos/builtin/packages/rocminfo/package.py [hunk 3]
```diff
@@ -128,12 +130,13 @@ class Rocminfo(CMakePackage):
         "5.4.3",
         "5.5.0",
         "5.5.1",
+        "5.6.0",
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/rocminfo/package.py:134]: Same question about `5.4.4`.

**Review by tldahlgren — COMMENTED** [review]
Confirmed the 17 version sha256.  Just wondering if there may be inconsistent addition and use of version `5.4.4` to the packages.

**Comment by dguibert** [comment]
> Just wondering if there may be inconsistent addition and use of version `5.4.4` to the packages.

the updated commit fixes the use of version 5.4.4 for all packages.
Thanks for noticing.

**Comment by srekolam** [comment]
As far as i remember, there is no 5.4.4 release for wider public distribution. i looked at https://repo.radeon.com/rocm/ and do not find the 5.4.4 release.  Also for 5.6.0, we have one PR #39673 that is just been submitted. request to see if the changes in that PR can be leveraged.

**Comment by dguibert** [comment]
> As far as i remember, there is no 5.4.4 release for wider public distribution. i looked at https://repo.radeon.com/rocm/ and do not find the 5.4.4 release.

You are right indeed, the tag exists on every repos but no "official" release in https://rocm.docs.amd.com/en/latest/CHANGELOG.html.

I'll test PR https://github.com/spack/spack/pull/39673, maybe updating rocprofiler if possible.

@renjithravindrankannath, we could update right thru 5.6.1 (https://rocm.docs.amd.com/en/latest/release.html)

**Comment by dguibert** [comment]
> As far as i remember, there is no 5.4.4 release for wider public distribution. i looked at https://repo.radeon.com/rocm/ and do not find the 5.4.4 release.

You are right indeed, the tag exists on every repos but no "official" release in https://rocm.docs.amd.com/en/latest/CHANGELOG.html.

I've tested PR https://github.com/spack/spack/pull/39673, hip@5.6.0 builds successfully, so will close this one.

@renjithravindrankannath, we could update right thru 5.6.1 (https://rocm.docs.amd.com/en/latest/release.html) and maybe updating rocprofiler if possible.

**Pull request closed without merging**